src/structures/Guild.js

```diff
@@ -504,6 +504,22 @@ class Guild {
   }
 
   /**
+   * Adds a bot user to the Guild.
+   * <warn>This is only available for user accounts, not bot accounts!</warn>
+   * @param {string} clientID The Client ID of the bot, this isn't always the bot's User ID
+   * @param {PermissionResolvable[]|number} [permissions=0] An optional array of PermissionResolvables or
+   * permissions number to give the bot upon authorization
+   * @returns {Promise}
+   */
+  addBot(clientID, permissions = 0) {
```

Comment:
change to something more representative of the actual action happening, like authenticateBot or authorizeBot something